pkg/local_object_storage/blobstor/common/storage.go

```diff
@@ -26,6 +28,12 @@ type Storage interface {
 	Put(PutPrm) (PutRes, error)
 	Delete(DeletePrm) (DeleteRes, error)
 	Iterate(IteratePrm) (IterateRes, error)
+
+	// OpenObjectStream looks up for referenced object in the Storage and, if the
+	// object exists, opens and returns stream with binary-encoded object. Returns
+	// [apistatus.ErrObjectNotFound] if object was not found. Resulting stream must
+	// be finally closed.
+	OpenObjectStream(oid.Address) (io.ReadSeekCloser, error)
```

Comment:
cosmetic: dont you think "stream" should be escaped here? "Reader" or smth more standard for `io` pkg looks more suitable for the disk (almost always) storages to me

Reply:
gone